README.md

```diff
@@ -27,142 +27,166 @@ The diagram below illustrates high-level design of each component as well as bas
 
 ![Architecture diagram](./assets/architecture.png)
 
-## Usage
+## Installation
 
-Before you can build and run the Miden node or any of its components, you'll need to make sure you have Rust [installed](https://www.rust-lang.org/tools/install). Miden node v0.2 requires Rust version **1.78** or later.
+The node software can be installed as a Debian package or using Rust's package manager `cargo`.
 
-Depending on the platform, you may need to install additional libraries. For example, on Ubuntu 22.04 the following command ensures that all required libraries are installed.
+Official releases are available as debian packages which can be found under our [releases](https://github.com/0xPolygonMiden/miden-node/releases) page.
 
-```sh
-sudo apt install llvm clang bindgen pkg-config libssl-dev libsqlite3-dev
-```
+Alternatively, the Rust package manager `cargo` can be used to install on non-debian distributions or to compile from source.
 
-### Installing the node
+### Debian package
 
-To install for production use cases, run:
+Debian packages are available and are the fastest way to install the node on Debian based system. Currently only `amd64` architectures are supported.
 
-```sh
-cargo install --path bin/node
-```
+These packages can be found under our [releases](https://github.com/0xPolygonMiden/miden-node/releases) page along with a checksum.
 
-This will install the executable `miden-node` in your PATH, at `~/.cargo/bin/miden-node`.
+Note that this includes a `systemd` service called `miden-node` (disabled by default).
 
-Otherwise, if only to try the node out for testing, run:
+To install, download the desired releases `.deb` package and checksum files. Install using
```

Comment:
Should we mention here (or somewhere else) that the user needs to be `miden`? Or is this not a requirement?

Reply:
afaict its not a requirement - the `miden` user is created as part of the post-install instructions. Which I initially found alarming but is apparently industry practice.. go figure.